lib/ConstPlugin.js

```diff
@@ -420,7 +420,10 @@ class ConstPlugin {
 								//
 								// ------------------------------------------
 								//
-								const dep = new ConstDependency(" undefined", expr.range);
+								const dep = new ConstDependency(
+									evaluated.isNull() ? " null" : " undefined",
```

Comment:
as I understand from babel https://babeljs.io/repl#?browsers=chrome%20%3E%2088&build=&builtIns=false&spec=true&loose=true&code_lz=MYewdgzgLgBAhjAvDA3gXwNwCgtwHQBGA_HnEA&debug=false&forceAllTransforms=false&shippedProposals=true&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Ces2015%2Creact%2Cstage-2%2Ctypescript%2Cenv&prettier=false&targets=Node-12&version=7.11.1&externalPlugins=

undefined should be used

Reply:
btw `"void 0".length < "undefined".length`

Reply:
interesting... I'll fix that